From bbb7444e77451d7e456281655b1102c6da279467 Mon Sep 17 00:00:00 2001
From: Rob Johnston <rob.johnston@joyent.com>
Date: Mon, 8 Apr 2019 18:44:23 +0000
Subject: [PATCH] OS-7309 build tools should support loader-based boot images
 (fix clean target) Reviewed by: John Levon <john.levon@joyent.com> Approved
 by: John Levon <john.levon@joyent.com>

---
 tools/format_image/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/format_image/Makefile b/tools/format_image/Makefile
index 9ba19826..e8751be7 100644
--- a/tools/format_image/Makefile
+++ b/tools/format_image/Makefile
@@ -37,4 +37,4 @@ format_image: $(OBJS)
 	$(CC) -c $(CFLAGS) $(CPPFLAGS) -o $@ $<
 
 clean:
-	rm $(TARGETS) $(OBJS)
+	rm -f $(TARGETS) $(OBJS)
-- 
2.21.0

